changelog.asc

```diff
@@ -2,6 +2,40 @@
 :sectanchors:
 = AnkiDroid Changelog
 
+== Version 2.20.0 (TBC)
+
```

Comment:
what about a tiny bit of humor since we did just pop the changelog a bunch with a .1, .2 and .3 rapid fire

```suggestion
Are you tired of updates yet? I hope not because we have a big one for you!

```

take it or leave, it, was an idle thought

Reply:
Leave it, unless there's a strong opinion 

Users are busy, and two (short) paragraphs feels like asking too much when they want to get on with their reviews